### PR TITLE
Changing the version of DistributedAvailabilityGroups  API for Azure CLI development

### PR DESCRIPTION
**Pull request by jovancevic123**
Readme Tag: package-composite-v5
Followed this TSG to get a private package for Azure CLI development purposes: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/private_package/get_private_package.md

# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Comment by jovancevic123** [comment]
I need only private package for now, after I finish development I will confirm that it is okay to publish this so that it can be used by Azure CLI publicly.

**Comment by jovancevic123** [comment]
I need to update tooling to use this. As this is a breaking change the tooling release date is Ignite release date.
When I request SDK release how can I be sure that everything will be on time? And do I need to create SDK release ASAP or to wait for November?

**Review by ericshape — APPROVED** [review]
LGTM but left a comment

File: specification/sql/resource-manager/readme.md [hunk 1]
```diff
@@ -1393,7 +1393,7 @@ input-file:
 - ./Microsoft.Sql/preview/2021-05-01-preview/LongTermRetentionManagedInstanceBackups.json
 - ./Microsoft.Sql/preview/2021-05-01-preview/RestorableDroppedManagedDatabases.json
 - ./Microsoft.Sql/preview/2021-05-01-preview/ServerConnectionPolicies.json
-- ./Microsoft.Sql/preview/2021-11-01-preview/DistributedAvailabilityGroups.json
+- ./Microsoft.Sql/preview/2023-08-01-preview/DistributedAvailabilityGroups.json
```
[the remark this answers is not in the document]
Comment (jovancevic123) [reply specification/sql/resource-manager/readme.md]: Done. Is it okay to merge this now, so I can request Python SDK before Ignite comes?
I need to use this in Azure CLI, that's why I need SDK.

**Review by rkmanda — APPROVED** [review]
:shipit:

**Pull request merged**